payments/mobile-pay/after-payment.md

```diff
@@ -32,66 +32,20 @@ sidebar:
 The `captures` resource lists the capture transactions performed on a
 specific payment.
 
-{:.code-header}
-**Request**
-
-```http
-GET /psp/mobilepay/payments/e7919b4f-81a2-4ffb-ec40-08d617d580a2/captures HTTP/1.1
-Host: {{ page.apiUrl }}
-Authorization: Bearer <AccessToken>
-Content-Type: application/json
-```
-
-{:.code-header}
-**Response**
+{% include transaction-response.md payment-instrument="mobile-pay" transaction="capture" %}
```

Comment:
Shouldn't it be `mobilepay` and not `mobile-pay` as the argument is used in API URLs where `mobilepay` is the correct spelling? Also, when the `transaction` is `capture`, the argument can be removed, no?

```suggestion
{% include transaction-response.md payment-instrument="mobilepay" %}
```